0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/README.md

```diff
@@ -1,8 +1,17 @@
 # Median Correlation Analysis
-In this analysis, the median values across all features were determined for all cells in each well. The aggreated features were then compared across replicates:
+In this analysis, the median values across all features were determined for all cells in each well. The aggreated features were then compared across groups:
 ### Correlation Comparisons
 Aggregated pairwise correlations were calculated across well between:
 - The same siRNA constructs within each construct concentration
 - Different siRNA constructs within each construct concentration
 
-Correlations were then plotted and compared by estimating the pdfs using kde for each construct concentration for the above groups.
+The `no_treatment` wells were only compared with other `no_treatment` wells for this analysis.
+
+### Visualzing Comparisons
+Correlations were then plotted as boxplots for each concentration in the following groups:
+
+- (Construct 1) and (Construct 2)
+- (Construct 1) and (Constrcut 1)
```

Comment:
How are these different? Shouldn't they have the exact same correlations? I am seeing the boxplots below are different, and I do not understand why

Reply:
Construct 1 and Construct 2 should have different off target effects, so I would expect the comparisons between respective features to vary more compared to the same constructs, leading to some difference between the correlation distributions, although, this also depends on how much the well conditions vary.

Reply:
Gotcha! I think I initially read the comparisons wrong.

I thought that there were two different boxplots for the following groups:

1. Construct 1 vs. Construct 2
2. Construct 2 vs. Construct 1

Theoretically, these should have the exact same values, if I'm understanding your procedure.

I think we're all clear now though! I will go ahead and merge this PR, and then update the figure in #11 

Thanks!